composer.json

```diff
@@ -17,6 +17,7 @@
         "php": "^5.5||^7.0",
         "cilex/console-service-provider": "~1.0",
         "doctrine/dbal": "~2.2",
+        "doctrine/annotations": "~1.2.0",
```

Comment:
doctine/annotations 1.3.0 requires PHP 5.6+, so we need to lock to an older version that still supports PHP 5.5